explorer/testdata/namespace/fail_unknown_member.carbon

```diff
@@ -11,6 +11,6 @@ package ExplorerTest api;
 namespace N;
 
 fn Main() -> i32 {
-  // CHECK:STDERR: COMPILATION ERROR: {{.*}}/explorer/testdata/namespace/fail_unknown_member.carbon:[[@LINE+1]]: member access into namespace is not implemented yet
+  // CHECK:STDERR: COMPILATION ERROR: {{.*}}/explorer/testdata/namespace/fail_unknown_member.carbon:[[@LINE+1]]: could not resolve 'value' in this scope
```

Comment:
"this scope" versus "namespace N" with PrintID?

Reply:
Seems reasonable, I'll send you a follow-up PR.